docs/usage/security-and-permissions.md

```diff
@@ -0,0 +1,56 @@
+# Security and Permissions
+
+## Comparison table
+
+Refer to this table for a overview of the permissions needed for the different ways to run Renovate:
+
+| Permission name   | Renovate hosted app |  Forking Renovate  |
+| ----------------- | :-----------------: | :----------------: |
+| Dependabot alerts |       `read`        |   not applicable   |
+| Administration    |       `read`        |       `read`       |
+| Metadata          |       `read`        |       `read`       |
+| Checks            | `read` and `write`  |   not applicable   |
+| Code              | `read` and `write`  |       `read`       |
+| Commit statuses   | `read` and `write`  | `read` and `write` |
+| Issues            | `read` and `write`  | `read` and `write` |
+| Pull Requests     | `read` and `write`  | `read` and `write` |
+| Workflows         | `read` and `write`  |   not applicable   |
+| Deployments       |   not applicable    |       `read`       |
+
+## List of permissions
+
+### Renovate hosted app
+
+The Renovate hosted app needs the following permissions.
```

Comment:
I don't know enough to say _why_ each app needs certain permissions. So help wanted on this. 😉